src/components/Pokemon.js

```diff
@@ -0,0 +1,39 @@
+import React, {useState, useEffect} from 'react';
+import axios from "axios";
+
+function Pokemon({name}) {
+    const [pokemonData, setPokemonData] = useState({});
+
+    useEffect(() => {
+        async function fetchPokemon() {
+            try {
+                const result = await axios.get(`https://pokeapi.co/api/v2/pokemon/${name}`);
+                setPokemonData(result);
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        fetchPokemon();
+    }, []);
+
+    return (
+        <>
+            {Object.keys(pokemonData).length > 0 &&
```

Comment:
Deze check wordt dan ook simpeler
```suggestion
            {pokemonData &&
```